src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMembersAttribute.cs

```diff
@@ -19,7 +19,7 @@ namespace System.Diagnostics.CodeAnalysis
     /// </remarks>
     [AttributeUsage(
         AttributeTargets.Field | AttributeTargets.ReturnValue | AttributeTargets.GenericParameter |
-        AttributeTargets.Parameter | AttributeTargets.Property,
+        AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.Method,
```

Comment:
We should put comments around this explaining why we need this in CoreLib but we don't want this to be exposed through contracts.

Reply:
Agreed.

One concern I have is if/when someone regenerates the `ref` assembly, this new AttributeTarget will get generated into the `ref` assembly, and it might not get caught during code review.

Reply:
+1 We want to be getting rid of the differences between autogenerated `ref` assembly, not adding new ones.